src/pages/List.js

```diff
@@ -1,30 +1,80 @@
 import dayjs from 'dayjs';
 import React, { useContext, useState } from 'react';
+import normalizeName from '../lib/normalizeName';
+import useListToken, { getCurrentToken } from '../useListToken';
 import { FirestoreCollection } from 'react-firestore';
-
 import { ListContext } from '../listContext';
-import useListToken, { getCurrentToken } from '../useListToken';
-import NavTabs from '../components/NavTabs';
-import Loading from '../components/Loading';
-import normalizeName from '../lib/normalizeName';
+import { Link } from 'react-router-dom';
+import AppHeader from '../components/AppHeader';
+import {
+  Container,
+  Loader,
+  Dimmer,
+  Input,
+  Accordion,
+  Checkbox,
+  Segment,
+  Icon,
+  Menu,
+  Responsive,
+  Button,
+  List as ListUI,
+} from 'semantic-ui-react';
 
 import './List.css';
-
-function isLessThan24hrs(datePurchased) {
-  return dayjs().diff(dayjs(datePurchased), 'hours') <= 24;
-}
-
-//we are checking if the last date it was purchased is less than 24hrs using isLessThan24hrs function
-function isChecked(lastDatePurchased) {
-  return !!lastDatePurchased && isLessThan24hrs(lastDatePurchased);
-}
+import NavTabs from '../components/NavTabs';
 
 const List = props => {
+  const {
+    shoppingList,
+    setShoppingList,
+    deleteItem,
+    purchaseItem,
```

Comment:
@stevelikesmusic  I'm getting an error that `purchaseItem` is not a function. I can't find it in listContext. Did it get lost somewhere when I was merging?

Reply:
![image](https://user-images.githubusercontent.com/3941856/74069809-22805c80-49b4-11ea-8646-f115dbd14272.png)

Reply:
Looks like you got this one ironed out @prophen. Seems like it was a funky merge?